packages/website/content/integrations/react-hook-form.mdx

```diff
@@ -4,186 +4,41 @@ slug: '/integrations/react-hook-form/'
-        <TextInput {...register('name', { required: true })} />
-
-        {/* Show a ValidationMessage if the input has any errors */}
-        {errors.name && (
-          <FormControl.ValidationMessage>
-            This field is required
-          </FormControl.ValidationMessage>
-        )}
-      </FormControl>
-
-      <FormControl isInvalid={Boolean(errors.description)}>
-        <FormControl.Label>Description</FormControl.Label>
-
-        {/* Registering a Textarea is similar to TextInput */}
-        <Textarea {...register('description', { maxLength: 20 })} />
-
-        {errors.description && (
-          <FormControl.ValidationMessage>
-            Your description must have a maximum of 20 characters
-          </FormControl.ValidationMessage>
-        )}
-      </FormControl>
-
-      <FormControl>
-        <FormControl.Label>City</FormControl.Label>
-
-        {/* When using a Select, it’s only necessary to register the Select component */}
-        <Select defaultValue="Mumbai" {...register('city')}>
-          <Select.Option value="Cape Town">Cape Town</Select.Option>
-          <Select.Option value="Mumbai">Mumbai</Select.Option>
-          <Select.Option value="Rio de Janeiro">Rio de Janeiro</Select.Option>
-        </Select>
-      </FormControl>
-
-      <FormControl as="fieldset">
-        <FormControl.Label as="legend">Martial status</FormControl.Label>
-
-        {/*
-          When using Radio.Group, it’s necessary to register each Radio with the same "name".
-          This is necessary because of how react-hook-form works.
-        */}
-        <Radio.Group defaultValue="single">
-          <Radio value="married" {...register('martial-status')} >
-            Married
-          </Radio>
-          <Radio value="single" {...register('martial-status')}>
-            Single
-          </Radio>
-        </Radio.Group>
-      </FormControl>
-
-      <FormControl isInvalid={Boolean(errors.checkbox)}>
-        <Checkbox {...register('checkbox', { required: true })}>
-          I confirm everything that said above is true
-        </Checkbox>
-
-        {errors.checkbox && (
-          <FormControl.ValidationMessage>
-            You must confirm that everything is true
-          </FormControl.ValidationMessage>
-        )}
-      </FormControl>
-
-      <Button variant="primary" type="submit">
-        Submit
-      </Button>
-    </Form>
-  );
-};
 ```
 
-## Integrating with `useController`
-
-First, it's necessary to create a controlled component which will use the `useController` hook and pass its props to the Forma 36 component.
-For example:
+## Examples
 
-```jsx static=true
-// ControlledTextInput.jsx
-import { useController } from 'react-hook-form';
-import { TextInput } from '@contentful/f36-forms';
+### Example of integration with `register()`
 
-export function ControlledTextInput(props) {
-  const {
-    field: { ref, ...inputProps },
-  } = useController(props);
+```jsx file=examples/RegisterIntegration.tsx
```

Comment:
@suevalov, how should I import 'react-hook-form'? It works in the playground, but the live preview on the documentation page doesn't see this package, there is the error `ReferenceError: useForm is not defined`
https://forma-36-git-hook-form-integration-ctfl.vercel.app/integrations/react-hook-form